src/screens/TopicPageScreen/index.js

```diff
@@ -32,11 +32,24 @@ import {getTopics, getUserTopic} from '../../service/topics';
 const styles = StyleSheet.create({
   parentContainer: {
     flex: 1
+  },
+  header: {
+    backgroundColor: 'lightblue',
+    justifyContent: 'center',
+    alignItems: 'center',
+    paddingTop: 40,
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    right: 0,
+    zIndex: 2
+  },
+  headerText: {
+    fontSize: 20,
+    fontWeight: 'bold'
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New styles for the header have been added. Make sure these styles are consistent with the overall design language of your application.
```diff
-   parentContainer: {
-     flex: 1
-   }
+   parentContainer: {
+     flex: 1
+   },
+   header: {
+     backgroundColor: 'lightblue',
+     justifyContent: 'center',
+     alignItems: 'center',
+     paddingTop: 40,
+     position: 'absolute',
+     top: 0,
+     left: 0,
+     right: 0,
+     zIndex: 2
+   },
+   headerText: {
+     fontSize: 20,
+     fontWeight: 'bold'
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->